docs/docs/aleph_zero/sdk.md

```diff
@@ -47,25 +71,13 @@ SDK also contains all essential values and helper functions that might be needed
  ┗ 📂tests
 ```
 
-### Contracts
-
-Contracts folder contains deploy-ready contracts and metadata that is used to interact with them.
-
-### Math
-
-Math contains Rust structs and data exported to WASM using wasm_bindgen which allowed use to the same functions and data that is used in main Invariant contract.
+## Usage Guide
 
-### Source
-
-Everything combined in easy-to-use interface.
-
-### Tests
-
-Tests folder with tests to make sure everything works right.
+Follow a step-by-step example demonstrating how to effectively use the Invariant SDK, with each step accompanied by code snippets. The complete code for these examples is available [here](https://google.com), ensuring a hands-on and comprehensive understanding of the SDK's functionality.
 
-## Usage
+### Select Network
```

Comment:
IMO we should split this section up a bit: "Select Network", "Initialize API", "Transaction signer"